### ISSUE TITLE
MDI icon not showing of showing incorrect

### ISSUE THREAD

**Issue by Zhaobab** [estevez-dev/ha_client#281]
Hi !

I try this new APP since 2 days, It's really awesome keep going guys !
I try to use custom icon for entity card (Lovelace UI) but app still use default:

```
entity: switch.plug_158d0001700170
hold_action:
  action: more-info
name: Veilleuse
tap_action:
  action: toggle
theme: Backend-selected
type: entity-button
icon: 'mdi:lightbulb'
```

Same for mdi:robot-vacuum ( who doesn't show at all).

WebUI :
![wechatimg108](https://user-images.githubusercontent.com/6453724/51822658-f6012180-2317-11e9-9a20-07ac83fd3299.jpeg)

App :
![wechatimg107](https://user-images.githubusercontent.com/6453724/51822668-fdc0c600-2317-11e9-8873-c5a4c3f8403e.png)

**Comment by Zhaobab** [comment]
> Vacuum is not yet supported. Contrary of other app this app is particularly well done. You can see the big developer effort on giving a, almost, faultless app. I prefer his approach (everything working) to other developers approach of bringing tons of features but full of bugs

I'm 100% agree with you guy but it wasn't my point to say Vacuum is not working. It seems to be a bug who occurs on basic cards like entity-button card.

Btw we all want a perfect app for our HA setup, and bug needs to be reported so this is my small contribution. Don't trigger like this

**Comment by estevez-dev** [comment]
Yes, the issue not in unsupported vacuums, but in icons in general

**Comment by Zhaobab** [comment]
> @Zhaobab I can't reproduce this issue. Are you sure you showed me the right config? Please show the config for robot vacuum entity button as well. Also check your app version please.

I was showing you the switch button at first but vacuum icon also have issue.
Here is the all view raw code :
`views:
  - badges:
      - sensor.yr_symbol
      - sun.sun
    cards:
      - entities:
          - vacuum.mivaccum
        show_header_toggle: false
        title: Aspirateur
        type: entities
      - entities:
          - entity: fan.xiaomi_air_purifier
        show_header_toggle: false
        title: Humidifier
        type: entities
      - entity: script.pushover_test
        hold_action:
          action: none
        icon: 'mdi:bell-ring'
        tap_action:
          action: toggle
        type: entity-button
      - type: entity-button
        tap_action:
          action: call-service
          service: vacuum.start
        hold_action:
          action: none
        entity: vacuum.mivaccum
        name: Nettoyage !
        icon: 'mdi:robot-vacuum'
    path: default_view
    title: Home`

App is 0.3.13 version
Screenshots:
![wechat image_20190129220325](https://user-images.githubusercontent.com/6453724/51913584-cdf5e900-2411-11e9-94da-329194522534.jpg)
![wechat image_20190129220330](https://user-images.githubusercontent.com/6453724/51913585-ce8e7f80-2411-11e9-8667-5f75dbe24f17.png)
![wechat image_20190129220335](https://user-images.githubusercontent.com/6453724/51913586-cf271600-2411-11e9-8ce3-338c68ef1a66.png)

**Comment by Zhaobab** [comment]
I don't know if bug is linked but script custom icon ```'mdi:bell-ring'``` also displaying default script icon

**Comment by estevez-dev** [comment]
Looks like any icon for entity-button is not working for you. What version of Home Assistant do you use?

**Comment by Zhaobab** [comment]
0.86.3 version

**Comment by estevez-dev** [comment]
Oh, I see. It was fixed after last release. So the fix is coming in app version 0.3.14
